lib/debezium/converters/time_test.go

```diff
@@ -43,3 +43,26 @@ func TestConvertDateTimeWithTimezone(t *testing.T) {
 		assert.Equal(t, expectedExtTime, val)
 	}
 }
+
+func TestConvertTimeWithTimezone(t *testing.T) {
+	{
+		// Invalid
+		ts, err := ConvertTimeWithTimezone("23:02")
+		assert.Nil(t, ts)
+		assert.ErrorContains(t, err, `failed to parse "23:02": parsing time`)
+	}
+	{
+		// What Debezium + Reader would produce
+		ts, err := ConvertTimeWithTimezone("23:02:06.745116Z")
```

Comment:
Can we add a test for a non-UTC time zone?

Reply:
Err sorry I meant can we add a test for a different time zone other than UTC, unless we only support parsing UTC?

Reply:
Added test case